test/expect-tests/memo/memoize_tests.ml

```diff
@@ -536,18 +551,18 @@ let print_result arg res =
 
 let%expect_test "error handling and memo - sync" =
```

Comment:
The name is outdated and I wonder of the whole test may be outdated too.

Reply:
Indeed, there is an async version just bellow. Removing it and updating the name of the async version

Reply:
Great!